pkg/cmd/subcommand/install.go

```diff
@@ -203,26 +213,40 @@ func (i *Install) RunInstall(cl *k8s.Clientset, cmd *cobra.Command) error {
 		}
 	}
 
+	// Record the list of components
+	// that currently exist in the cluster.
+	if _, err := operator.GetInventoryRecord(ctx, false); err != nil {
+		return errors.Wrap(err, "failed to get inventory record")
+	}
+	defer operator.RecordInventory(ctx)
+
+	c := make(chan os.Signal)
+	signal.Notify(c, os.Interrupt, syscall.SIGTERM)
+	go func() {
+		<-c
```

Comment:
What if user never press `ctlr + c` ?

Reply:
This goroutine will stop when the main function has done.